### ISSUE TITLE
poetry add removes package

### ISSUE THREAD

**Issue by wkornewald** [python-poetry/poetry#1473]
- [x] I am on the [latest](https://github.com/sdispater/poetry/releases/latest) Poetry version.
- [x] I have searched the [issues](https://github.com/sdispater/poetry/issues) of this repo and believe that this is not a duplicate.
- [ ] If an exception occurs when executing a command, I executed it again in debug mode (`-vvv` option).

- **OS version and name**: Linux / Debian 10
- **Poetry version**: 0.12.17
- **Link of a [Gist](https://gist.github.com/) with the contents of your pyproject.toml file**: https://gist.github.com/wkornewald/a910b156944a364b457a6b1a91fa54f8

## Issue

Whenever I add a new package poetry removes grpcio, but when I run `poetry update` grpcio comes back. Sometimes `peotry update` doesn't work and in that case I delete `poetry.lock` and run `poetry lock` (maybe optional? haven't tested) and `poetry update` to get that package back.

```
root@...# poetry add redis
Using version ^3.3 for redis

Updating dependencies
Resolving dependencies... (1.3s)

Writing lock file


Package operations: 0 installs, 0 updates, 1 removal

  - Removing grpcio (1.24.1)
root@...# poetry update
Updating dependencies
Resolving dependencies... (4.4s)

Writing lock file


Package operations: 1 install, 0 updates, 0 removals

  - Installing grpcio (1.24.1)
```

**Comment by sdispater** [comment]
Could you check if the behavior is the same with the `1.0.0b2` release?

**Comment by wkornewald** [comment]
I think it works correctly on that version. Do you plan to backport the fix to a stable release or will 1.0 soon be released?
